extensibility.config.js

```diff
@@ -0,0 +1,40 @@
+export default {
```

Comment:
would be very nice to optionally enable telemetry too by just some config prop(s) here